src/sentry/static/sentry/app/components/sidebar/sidebarDropdown/index.jsx

```diff
@@ -1,243 +0,0 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import styled from '@emotion/styled';
-
-import {logout} from 'app/actionCreators/account';
-import {t} from 'app/locale';
-import Avatar from 'app/components/avatar';
-import ConfigStore from 'app/stores/configStore';
-import DropdownMenu from 'app/components/dropdownMenu';
-import Hook from 'app/components/hook';
-import IdBadge from 'app/components/idBadge';
-import InlineSvg from 'app/components/inlineSvg';
-import Link from 'app/components/links/link';
-import SentryTypes from 'app/sentryTypes';
-import SidebarDropdownMenu from 'app/components/sidebar/sidebarDropdownMenu.styled';
-import SidebarMenuItem, {menuItemStyles} from 'app/components/sidebar/sidebarMenuItem';
-import SidebarOrgSummary from 'app/components/sidebar/sidebarOrgSummary';
-import TextOverflow from 'app/components/textOverflow';
-import withApi from 'app/utils/withApi';
-
-import Divider from './divider.styled';
-import SwitchOrganization from './switchOrganization';
-
-const SidebarDropdown = withApi(
-  class SidebarDropdown extends React.Component {
-    static propTypes = {
-      api: PropTypes.object,
-      orientation: PropTypes.oneOf(['top', 'left']),
-      collapsed: PropTypes.bool,
-      org: SentryTypes.Organization,
-      user: SentryTypes.User,
-      config: SentryTypes.Config,
-      onClick: PropTypes.func,
-    };
-
-    static defaultProps = {
-      onClick: () => {},
```

Comment:
I removed the onClick because it actually wasn't being used correctly. Avatar doesn't take a onClick.

(I did fix Avatar in a diff PR to take on click, but then I couldn't get the text to take an onClick in this component, so only the avatar was triggering the onClick which is wrong. The onClick itself is used to hide the sidebar panel when the menu is opened. Not super important. I may revisit later)